docs/Compatibility.md

```diff
@@ -4,156 +4,247 @@ The Ruby Datadog Trace library is open source. See the [dd-trace-rb][1] GitHub r
+
 [38]: https://docs.datadoghq.com/tracing/trace_collection/dd_libraries/ruby#redis
+
 [39]: https://docs.datadoghq.com/tracing/trace_collection/dd_libraries/ruby#resque
+
 [40]: https://docs.datadoghq.com/tracing/trace_collection/dd_libraries/ruby#rest-client
+
 [41]: https://docs.datadoghq.com/tracing/trace_collection/dd_libraries/ruby#roda
+
 [42]: https://docs.datadoghq.com/tracing/trace_collection/dd_libraries/ruby#sequel
+
 [43]: https://docs.datadoghq.com/tracing/trace_collection/dd_libraries/ruby#shoryuken
+
 [44]: https://docs.datadoghq.com/tracing/trace_collection/dd_libraries/ruby#sidekiq
+
 [45]: https://docs.datadoghq.com/tracing/trace_collection/dd_libraries/ruby#sinatra
+
 [46]: https://docs.datadoghq.com/tracing/trace_collection/dd_libraries/ruby#sneakers
+
 [47]: https://docs.datadoghq.com/tracing/trace_collection/dd_libraries/ruby#stripe
+
 [48]: https://docs.datadoghq.com/tracing/trace_collection/dd_libraries/ruby#sucker-punch
 
+### Support Policy
+
+Datadog APM for Ruby is built upon dependencies defined in specific versions of the host operating system, Ruby
+runtime, certain Ruby libraries, and the Datadog Agent/API. When these versions are no longer supported by their
+maintainers, Datadog APM for Ruby limits its support for these as well.
+
+#### Levels of Support
+
+| **Level**                                              | **Support provided**                                                                                                                                                         |
+|--------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| <span id="support-unsupported">Unsupported</span>      | No implementation. [Contact our customer support team for special requests.](https://www.datadoghq.com/support/)                                                             |
+| <span id="support-beta">Beta</span>                    | Initial implementation. May not yet contain all features. Support for new features, bug & security fixes provided on a best-effort basis.                                    |
+| <span id="support-ga">General Availability (GA)</span> | Full implementation of all features. Full support for new features, bug & security fixes.                                                                                    |
+| <span id="support-maintenance">Maintenance</span>      | Full implementation of existing features. Does not receive new features. Support for critical bug & security fixes only.                                                     |
+| <span id="support-legacy">Legacy</span>                | Legacy implementation. May have limited function, but no maintenance provided. [Contact our customer support team for special requests.](https://www.datadoghq.com/support/) |
```

Comment:
Seems like `Legacy` != `Maintenance`, but I feel a bit confused as to how `Legacy` is distinct from `EOL`. It sounds like a LTS-upon-request-otherwise-EOL?

Reply:
`Legacy` is unimplemented; it was a postulated level of support. If not applicable, we might just want to delete it under the principle of `less is more`.

Reply:
@delner I think that makes sense. We could always re-introduce the tier if we ever wind up using it.

Reply:
Agreed, less source of confusion.